scripts/lint-with-types.sh

```diff
@@ -21,7 +16,7 @@ rest)
     yarn lerna run --ignore=$PRIMARY_PACKAGES --no-bail lint
     ;;
 *)
-    # all scopes
-    yarn lint
+    echo "The regular lint command now lints with types. Just use that."
+    exit 0
```

Comment:
Should we `exit 1` here, and echo to stderr ?

Reply:
I did mean to exit 1. nbd tho, this script was used only by CI and it's going away